service/src/main/java/com/myhome/controllers/ApiErrorHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.myhome.controllers;
+
+import com.myhome.controllers.response.ExceptionResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class ApiErrorHandler {
+
+  @ExceptionHandler(value = { Exception.class })
```

Comment:
Let's pass here all the exceptions that we throw in application and create handleException method per each HTTP code that we want to return - we will do not have to manage the http codes in exception descriptions.

Non handled exceptions should be handled as INTERNAL_SERVER_ERROR (500 code)